pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.black]
+line-length = 88
+include = '\.pyi?$'
+exclude = '''
+/(
+    \.git
+  | \.hg
+  | \.mypy_cache
+  | \.tox
+  | \.venv
+  | \.ve
+  | ve
+  | env
+  | _build
+  | buck-out
+  | build
+  | dist
+  | migrations
```

Comment:
This looks to be the default `black` options, but with `.direnv`, `.eggs`, `.nox`, `venv`, `.svn`, `.ipynb_checkpoints`, `__pypackages__` removed and `ve`, `env`, `migrations` added. Is there a reason for this? I usually have the migrations formatted and it doesn't seem to cause issues. Specifying `exclude` will also make black no longer look at gitignore for things to exclude, and gitignore is usually where we add things like `ve` and `env`.